src/lib/Form/Type/ChoiceList/Loader/ConfiguredLanguagesChoiceLoader.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Form\Type\ChoiceList\Loader;
+
+use eZ\Publish\API\Repository\LanguageService;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use Symfony\Component\Form\ChoiceList\ArrayChoiceList;
+use Symfony\Component\Form\ChoiceList\Loader\ChoiceLoaderInterface;
+
+class ConfiguredLanguagesChoiceLoader implements ChoiceLoaderInterface
+{
+    /** @var \eZ\Publish\API\Repository\LanguageService */
+    private $languageService;
+
+    /** @var string[] */
+    private $siteAccessLanguages;
+
+    /**
+     * @param \eZ\Publish\API\Repository\LanguageService $languageService
+     * @param \eZ\Publish\Core\MVC\ConfigResolverInterface $configResolver
+     */
+    public function __construct(LanguageService $languageService, ConfigResolverInterface $configResolver)
+    {
+        $this->languageService = $languageService;
+        $this->siteAccessLanguages = $configResolver->getParameter('languages');
```

Comment:
`$configResolver->getParameter('languages')` should be called in https://github.com/ezsystems/ezplatform-admin-ui/pull/1223/files#diff-5f4e37227ab6d0e555d4b6703d406632R96